lib/platform/node/node_http.js

```diff
@@ -85,6 +85,28 @@
             return req;
         },
 
+        makeRequestAsync: async function(url, message) {
```

Comment:
Just realized we are missing docstrings for these new methods - can we add those?

Reply:
Good catch again 😅
Changes done.